modules/framework/galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/FrameworkInitialisation.java

```diff
@@ -117,13 +108,23 @@ public FrameworkInitialisation(
             this.logger, overrideProperties, this.cpsFramework, this.fileSystem);
         this.dssFramework = initialiseDynamicStatusStore(logger,bundleContext);
 
-        if (testrun) {
-            // *** Is this a test run,
-            // *** Then we need to make sure we have a runname for the RAS. If there isnt
-            // one, we need to allocate one
-            // *** Need the DSS for this as the latest run number number is stored in there
-            String runName = locateRunName(this.cpsFramework);
-            this.framework.setTestRunName(runName);
+        switch(type) {
```

Comment:
Might be worth putting this into a separate method so the overall method doesn't get any bigger.